snuba/migrations/snuba_migrations/stacktraces/0001_stacktraces.py

```diff
@@ -0,0 +1,92 @@
+from typing import List, Sequence
+
+from snuba.clickhouse.columns import (
+    UUID,
+    Column,
+    DateTime,
+    String,
+    UInt,
+    NamedTuple,
+)
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations, table_engines
+from snuba.migrations.columns import MigrationModifiers as Modifiers
+
+columns: List[Column[Modifiers]] = [
+    # primary key
+    Column("organization_id", UInt(64)),
+    Column("project_id", UInt(64)),
+    Column("transaction_id", UUID()),
+    Column("received", DateTime()),
+
+    # profiling data
+    Column("stacktrace", String(Modifiers(codecs=["LZ4HC(9)"]))),
+    Column("symbols", String(Modifiers(codecs=["LZ4HC(9)"]))),
+
+    # filtering data
+    Column("android_api_level", UInt(32, Modifiers(nullable=True))),
+    Column("device_classification", String(Modifiers(low_cardinality=True))),
```

Comment:
Yes, and it actually can be an `Enum` since we classify devices in 3 values (`high`/`mid`/`low`).

Reply:
if it can be an enum, please keep using lowcardinality. The storage underneath performs in the same way for all practical purposes and you do not haveto hardcode values in the schema.